sklearnserver/rockcraft.yaml

```diff
@@ -0,0 +1,78 @@
+name: sklearnserver
+summary: An image for Seldon SKLearn Server
+description: |
+  This image is used as part of the Charmed Kubeflow product. The SKLearn Server serves
+  models which have been stored as pickles.
+version: v1.16.0_1 # version format: <KF-upstream-version>_<Charmed-KF-version>
+license: Apache-2.0
+base: ubuntu:20.04
+services:
+  sklearnserver:
+    override: replace
+    summary: "sklearnserver service"
+    startup: enabled
+    # Yet again, use a subshell to jam conda into a working state. Can't use bashrc, because it immediately
+    # exits if PS1 isn't set, so no-go from scripts
+    command: bash -c 'cd /microservice && export PATH=/opt/conda/bin/${PATH} && eval $(/opt/conda/bin/conda shell.bash hook 2> /dev/null) && source /opt/conda/etc/profile.d/conda.sh && conda activate && seldon-core-microservice $MODEL_NAME --service-type $SERVICE_TYPE --persistence $PERSISTENCE'
+    environment:
+      MODEL_NAME: "SKLearnServer"
+      SERVICE_TYPE: "MODEL"
+      PERSISTENCE: "0"
```

Comment:
I had the impression we don't need this anymore if we don't use s2i.

Reply:
While it's true that they're actually used [here](https://github.com/SeldonIO/seldon-core/blob/21f95a4b01ff1ce18bc2a4e7cb86c86698050fa3/wrappers/s2i/python/s2i/bin/run#L46), they are actually passed in as arts to `seldon-core-microservice`, and making them available in the environment ensures that they're set in a nicer way than jamming them into the `command` block, at least.